test/unit/signin.test.ts

```diff
@@ -53,6 +54,29 @@ describe('signin', () => {
     restoreOriginalFetch();
   });
 
+  test('getSigninRequestHeaders', () => {
+    expect(getSigninRequestHeaders({})).toEqual({});
+
+    const accessToken = 'someToken';
+    expect(getSigninRequestHeaders({ accessToken })).toEqual({ [HEADER_X_ACROLINX_AUTH]: accessToken });
+
+    const username = 'name§$%&/(';
```

Comment:
Please add test with space, dot(.) and @ in username.

Reply:
done 👍🏻